.github/workflows/java-continuous-integration.yml

```diff
@@ -0,0 +1,127 @@
+# Description
+# ===========
+# This workflow is triggered each time
+# commits are pushed to GitHub or a pull request is opened.
+# It launches three jobs in parallel : a build with java 8,
+# a build with java 11 and a SonarCloud analysis.
+---
+name: Java CI
+
+on: [push, pull_request]
+
+env:
+  SONARQUBE_VERSION: 8.9.0-community 
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ '8', '11' ]
+    name: Java ${{ matrix.Java }} CI
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Setup java
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'adopt'
+          java-version: ${{ matrix.java }}
+      - name: Cache Maven packages
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-m2
+      - name: Cache node_modules
+        uses: actions/cache@v2
+        with:
+          path: node_modules
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: ${{ runner.os }}-yarn-
+      - name: Build with Maven
+        run: mvn -B clean package
+      - name: Test cnes-report
+        env:
+          SONARQUBE_VERSION: 8.9.0-community
+        run: |
+          version=$(cat pom.xml | grep "<version>.*</version>" | head -1 |awk -F'[><]' '{print $3}');
+          echo "Starting docker";
+          docker run --name sonarqube_${SONARQUBE_VERSION} -d -p 9000:9000 sonarqube:${SONARQUBE_VERSION};
+          echo "Inject plugin";
+          docker cp target/sonar-cnes-report-${version}.jar sonarqube_${SONARQUBE_VERSION}:/opt/sonarqube/extensions/plugins/;
+          docker exec -u root sonarqube_${SONARQUBE_VERSION} chown sonarqube:sonarqube /opt/sonarqube/extensions/plugins/sonar-cnes-report-${version}.jar;
+          docker restart sonarqube_${SONARQUBE_VERSION};
+          echo "Waiting up to 5 minutes for SonarQube...";
+          counter=0;
+          limit=300;
+          status_sonar=$(curl -s "http://localhost:9000/api/system/status" | grep "\"status\":\"UP\"" > /dev/null; echo $?);
+          while [[ 0 -ne $status_sonar && $counter -le $limit ]]; do
+              sleep 1;
+              counter=$(( $counter + 1 ));
+              status_sonar=$(curl -s "http://localhost:9000/api/system/status" | grep "\"status\":\"UP\"" > /dev/null; echo $?);
+          done;
+          mvn sonar:sonar -Dsonar.host.url=http://localhost:9000 -Dsonar.login=admin -Dsonar.password=admin -Dsonar.organization=default-organization;
+          ce=$(grep ceTaskUrl= target/sonar/report-task.txt);
+          ceTaskUrl=${ce:10};
+          continue=true;
+          while [ $continue = true ]; do
```

Comment:
Could please add a comment or a simple "echo" explaining we are waiting for the SonarQube data ingestion to complete here ?